azext_iot/product/shared.py

```diff
@@ -44,4 +44,4 @@ class ValidationType(Enum):
     certification = "Certification"
 
 
-BASE_URL = "https://certify.azureiotsolutions.com"
+BASE_URL = "https://prod.certsvc.trafficmanager.net"
```

Comment:
Is this URL publicly documented? Interesting there is no azure branding.

Reply:
Traffic manager is an Azure product https://docs.microsoft.com/en-us/azure/traffic-manager/traffic-manager-overview